src/ext/java/org/rutebanken/tiamat/ext/fintraffic/auth/AuthorizationException.java

```diff
@@ -0,0 +1,7 @@
+package org.rutebanken.tiamat.ext.fintraffic.auth;
+
+public class AuthorizationException extends RuntimeException {
```

Comment:
The base exception types should probably be a bit less generic?

Reply:
Probably the most sensible solution would be to let it bubble to frontend so that the user can see that there's something problematic going on as this implies a generic problem in the overall request/response handling when calling Trivore's Management API, and could be anything.